kong/conf_loader.lua

```diff
@@ -70,6 +70,7 @@ local CONF_INFERENCES = {
   latency_tokens = {typ = "boolean"},
   error_default_type = {enum = {"application/json", "application/xml",
                                 "text/html", "text/plain"}},
+  client_max_body_size = {typ = "string"},
```

Comment:
since `client_body_buffer_size` is a string, it isnt necessary to define it here. i can for explicitness if you like.